skins/oasis/modules/templates/Body_Index.php

```diff
@@ -67,6 +67,8 @@
 			}
 		?>
 
+		<?= !empty( $wg->AdDriverUseRevcontent ) ? $app->renderView( 'Ad', 'Index', ['slotName' => 'REVCONTENT_ABOVE_ARTICLE'] ) : '' ?>
```

Comment:
I think it will slow down the roadblock. We should create here empty div and push new slots (to `window.adslots2`) from JS on blocking event (`helper::addOnBlockingCallback`).